content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,115 @@
+title: "How Does Timecode Vinyl Actually Work? (Pt. 1)"
+authors: Jan Holthuis
+tags: 2.3, timecode, dvs, vinyl control, serato, cd
+status: draft
+
+
+Since the dawn of DJing, spinning vinyl records never really went out of fashion in the DJ community.
```

Comment:
The tense of this sentence is a little funny.  "Never went out of fashion" is present tense, whereas "since the dawn" is a span of time.  Better would be:  "Since the dawn of DJing, spinning vinyl records has never really gone out of fashion" ("in the DJ community" is redundant and unneeded)